index.html

```diff
@@ -223,7 +223,7 @@ <h2 class="language">Languages</h2>
         </li>
         <li class="about-card">
 
-          <img class="icon_language " src="Images/icon-frameworks.svg" alt=" language icon" />
+          <img class="icon_language " src="Images/icon-frameworks.svg" alt=" languages icon" />
```

Comment:
Kindly change the contrast to be at least `4.5` of the `See project` buttons in `hover` because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/160629299-124835f9-f8eb-46b9-9230-9e55e06037b1.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
- [ ] the Same problem still exists Please fix the contrast issue as the last reviewer mentioned and use the link he gave to you to test your changes. this [link](https://www.aditus.io/button-contrast-checker/).
Hint: Change the colors a little bit to get the right contrast ratio but don,t fade away so much from the given colors on the Figma design.

![image](https://user-images.githubusercontent.com/68796232/160636532-a46bcdf7-a4ba-4093-bd29-4797b82ae769.png)